astropy/utils/data.py

```diff
@@ -166,7 +172,9 @@ def _is_inside(path, parent_path):
 @contextlib.contextmanager
 def get_readable_fileobj(name_or_obj, encoding=None, cache=False,
                          show_progress=True, remote_timeout=None,
-                         sources=None, http_headers=None):
+                         sources=None, http_headers=None,
+                         use_fsspec=None, fsspec_kwargs=None,
```

Comment:
(unrelated to the PR): another showcase that it would be so nice to have kwarg only arguments for functions like this.

Reply:
Update: I have now modified this function to make the three new arguments keyword-only. (commit 6e7b6095f97e25d8491cd2ae0d61b2052dc19d3c)